mockgcp/mockgcptests/proxy.go

```diff
@@ -89,16 +94,22 @@ func (p *Proxy) BuildGcloudConfig(proxyEndpoint *net.TCPAddr, mockgcp mockgcp.In
 	config.AddConfig("proxy/port", strconv.Itoa(proxyEndpoint.Port))
 
 	// We need to register these over http, to stop gcloud trying to use TUNNEL with our proxy
-	services := mockgcp.ListServices()
+
+	// We need a hard-coded list, because we don't always mockgcp available
+	services := []string{
```

Comment:
Since we have more and more hard-coded lists in the MockGCP, we may want to start moving them to its own file `mockgcp/const` dir or at least to the top of this file, otherwise the maintenance overhead will continue grow dramatically.

Reply:
I sent a PR to just add all the services.  #3655   cc @cheftako